snmp/datadog_checks/snmp/config.py

```diff
@@ -156,7 +162,7 @@ def get_context_data(instance):
     @staticmethod
     def parse_metrics(metrics, enforce_constraints, warning):
```

Comment:
(nit) Could you add a doc string about the share of the object that is returned ? that could help readers.

Reply:
Done.